Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -4,6 +4,19 @@ import MapboxDirections
 
 let NavigationEventTypeRouteRetrieval = "mobile.performance_trace"
 
+/**
+ The `PassiveNavigationEventsManagerDataSource` protocol declares values required for recording passive location events.
```

Comment:
Nit: In general, it’s better to start a top-level type’s documentation by saying what it is, before saying what it does. For example:

> A data source that declares values required for recording passive location events.

In the Objective-C days, it was common to start a type’s documentation by saying the type’s name, but that’s no longer necessary. It doesn’t make a huge difference in this case, other than removing some redundancy. But in general it’s a good exercise, ensuring we think hard about what a type represents so we don’t wind up with abstract FactoryManagerCoordinators that are hard to reason about.